src/ops.jl

```diff
@@ -189,7 +184,7 @@ trace(A::FixedMatrix{2,2}) = A[1,1] + A[2,2]
 trace(A::FixedMatrix{3,3}) = A[1,1] + A[2,2] + A[3,3]
 trace(A::FixedMatrix{4,4}) = A[1,1] + A[2,2] + A[3,3] + A[4,4]
 
-\{m,n,T}(mat::Mat{m,n,T}, v::Vec{n, T}) = inv(mat)*v
+\{m,n,T1,T2}(mat::Mat{m,n,T1}, v::Vec{n,T2}) = inv(mat)*v
```

Comment:
I would say theoretically we don't now. Practically, that is up to test. There is http://arxiv.org/abs/1201.6035 and the fact that the computation of the inverse is a very "parallel" computation so maybe behaves nicely with instruction pipelines.